resources/teamcity/models/models-build-deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+
+# shellcheck disable=SC2164
+# shellcheck disable=SC1091
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+builder_describe \
+  "Build and deploy models" \
+  "all            run all actions" \
+  "build          build models" \
+  "publish        deploy models" \
+  "--downloads-keyman-com=DOWNLOADS_KEYMAN_COM_URL  URL of downloads server" \
+  "--rsync-path=REMOTE_RSYNC_PATH                   Path to rsync on remote server" \
+  "--rsync-dest=RSYNC_DEST                          Destination for rsync"
```

Comment:
* Do these variables need to be exported in order for them to be available to ci.sh?